src/atoms/Space/Space.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, text } from '@storybook/addon-knobs';
+import Text from '../Text';
+import Space from './Space';
+
+const sizeOptions = {
+  xxsmall: 'xxsmall',
+  xsmall: 'xsmall',
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+};
+
+storiesOf('Space', module)
+  .addParameters({
+    component: Space,
+  })
+  .add('default', () => (
+    <Space margin={[1, 1]} padding={[2, 2]}>
+      <Text size={select('Size', sizeOptions, 'medium')} as="span">
```

Comment:
Don't think size knob for Text component is useful for Space stories.
Better have knobs around Space component itself.